changelog.json

```
@@ -1,5 +1,19 @@
 {
   "releases": {
+    "2.2.4": [
+      "[New] Warn when committing to a protected branch - #7023",
+      "[Added] Adding integration for Xcode as external editor - #8255. Thanks @msztech!",
+      "[Fixed] Changed wording to match git autocrlf behavior - #5741. Thanks @jmhammock!",
+      "[Fixed] Don't dismiss dialogs when dragging from the dialog to the backdrop - #8544",
+      "[Fixed] Fix arrow key navigation of Changes and History lists (#6845) - #6845. Thanks @JMcKiern!",
+      "[Fixed] Handle multibyte characters in names - #8534",
+      "[Fixed] Keep PR badge on top of progress bar - #8622",
+      "[Fixed] Prevent application crash when background process spawn fails - #8614",
+      "[Fixed] Show autocomplete suggestions on top of CodeMirror author input - #4084",
+      "[Fixed] Standardize dialogs' button orders - #7186 #2463 #5883",
+      "[Improved] Make no newline indicator in file diffs readable by screenreaders. - #6494. Thanks @jmhammock!",
+      "[Improved] Warn user if they're committing to a repo they don't have write permission for - #8665"
```

Last thing, so sorry! I just saw this, should we make this [New]? It feels analogous to the protected branches one based on the way it’s written

---

Nothing to apologize for! In fact I think this is also a good opportunity to look at the language one more time. How about...

```suggestion
      "[New] Warning when committing to a repository you don't have write access to - #8665"
```

---

```suggestion
      "[New] Warn when committing to a repository you don't have write access to - #8665"
```